start_dev.sh

```diff
@@ -2,9 +2,17 @@
 
 # WARNING!! use this script while running without docker.
 
+#- 
+# Pull configuration info from /config.toml
+#
+# Read the values from the config.toml file
+source <(grep = config.toml | sed -E 's/ *= */=/g')
```

Comment:
I am probably doing something wrong, but seems like the env vars are not being populated, I recommend running: 

```
source config.toml
```

Reply:
Nope... looks like you are correct.  Super frustratingly that command (with grep|sed) works fine If I copy/paste as-is to the command line.  So does simply using `source config.toml`.  I am going to agree with you that the simpler way without sed is better.